include/boost/math/distributions/dirichlet.hpp

```diff
@@ -223,20 +199,20 @@ class dirichlet_distribution
   } // dirichlet_distribution constructor.
 
   // Get the concentration parameters.
-  RandomAccessContainer &alpha() const { return m_alpha; }
+  const RandomAccessContainer &get_alpha() const { return m_alpha; }
   // Set the concentration parameters.
-  RandomAccessContainer &alpha() { return m_alpha; }
+  RandomAccessContainer &set_alpha() { return m_alpha; }
```

Comment:
What is the use case for `set_alpha()`? Isn't `alpha` set in the constructor?

Reply:
It's not that important but provides a little flexibility. This helps in case ```m_alpha``` (concentration parameter) needs to be changed (or updated) on the go.

Reply:
If you don't have an applications that needs this, I would definitely remove it. In fact, `set_alpha` is more expensive than generating a new object since instantiation is done by move.

Reply:
No worries. I will remove it.